trunk/SUAVE/Analyses/Stability/Fidelity_Zero.py

```diff
@@ -122,8 +122,15 @@ def __call__(self,conditions):
             surf.ep_alpha = Supporting_Functions.ep_alpha(surf.CL_alpha, sref, span)
         
         # Static Stability Methods
-        static_stability.cm_alpha = taw_cmalpha(geometry,mach,conditions,configuration)
-        static_stability.cn_beta  = taw_cnbeta(geometry,conditions,configuration)
+        static_stability.Cm_alpha,static_stability.cm0, static_stability.CM  = taw_cmalpha(geometry,mach,conditions,configuration)
+	#print static_stability.cm_alpha[0][0]
+        if geometry.wings.has_key('vertical_stabilizer'):
+	    static_stability.cn_beta  = taw_cnbeta(geometry,conditions,configuration)
```

Comment:
Capitalization of Cn_beta is not consistent here. Also check indentation in this file.